src/pages/edit.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import Edit from "../components/Edit/Edit"
+import { Layout } from "../components/Layout/Layout"
+
+export default function EditPage({ location }: any) {
+  return (
+    <div className="section">
+      <div className="container is-fluid">
+        <Layout pageTitle="Photo Edit Page">
+          <Edit location={location.state.item} />
```

Comment:
the build is mean xD I will try to figure it out :)

Reply:
I think it's failing because the way that I passed the data here, if it's built statically it will be undefined. Not really sure how to fix this, but I'll look into it

Reply:
https://github.com/IMY772-2022/via-web/runs/6778765705?check_suite_focus=true

You can see the build log there, and why it is failing. But basically, I think the way you did it previously was not wrong, its just complaining because the `item` could be undefined